### PR TITLE
Update LNW documentation with LNW v2 details

### PR DESCRIPTION
**Pull request by n-sandeep**
Modify existing ECMP, FRR and IPv6 documentation with latest LNW v2 details.

### CODE REVIEW

File: docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+     "vmeta.common.vsi=9,target_vsi=27,action=linux_networking_control.fwd_to_vsi(43)"
+
+ p4rt-ctl add-entry br0 linux_networking_control.vsi_to_vsi_loopback \
+     "vmeta.common.vsi=27,target_vsi=9,action=linux_networking_control.fwd_to_vsi(25)"
+
+```
+
+### Configure rules for mapping between Physical port and ACC port representor
+
+Configure rules to send ingress packets from a physical port to its respective port representors.
```
Comment (ffoulkes) [inline docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md:226]:
```suggestion
Configure rules to send ingress packets from a physical port to its associated port representors.
```
Grammar error. _physical port_ is singular. _respective_ is used to map plural to plural.

File: docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+
+Refer above port mapping for physical port to ACC port representor mapping. Here sample commands are shown for a single physical port, configure similar mapping for remaining physical ports.
```
Comment (ffoulkes) [inline docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md:228]:
```suggestion
Refer to above port mapping for physical port to ACC port representor mapping. Here, sample commands are shown for a single physical port. Configure similar mapping for remaining physical ports.
```

1. _Refer to_, not _refer_. They have different meanings.
  - "Refer to X" instructs the reader to look elsewhere for information.
  - "Refer X to Y" means to tell person or persons X to go see person Y (or an example of _metonymy_, "a figure of speech that consists of the use of the name of one object or concept for that of another to which it is related, or of which it is a part").
2. Comma following disjunct adverb.
3. Comma splice. Start a new sentence.

File: docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+
+Example:
+
+- Physical port 0 port id is 0
+- Corresponding port representor VSI value 17
```
Comment (ffoulkes) [inline docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md:233]: I believe you're missing a verb.

File: docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md [hunk 4]
```diff
@@ -0,0 +1,456 @@
...
+
+ p4rt-ctl add-entry br0 linux_networking_control.tx_acc_vsi \
+     "vmeta.common.vsi=17,zero_padding=0,action=linux_networking_control.l2_fwd_and_bypass_bridge(0)"
+```
+
+### Configure rules for mapping between APF netdev on HOST and ACC port representor
+
+Configure rules to send APF netdev on HOST to its respective port representors.
+
+Refer above port mapping for APF netdev on HOST to ACC port representor mapping. Here sample commands are shown for APF netdev on HOST, configure similar mapping for remaining APF netdevs on HOST.
```
Comment (ffoulkes) [inline docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md:257]: Same comment as line 228.

File: docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md [hunk 5]
```diff
@@ -0,0 +1,456 @@
...
+
+Example:
+
+- APF netdev 1 on HOST has a VSI value 24
+- Corresponding port representor VSI value 18
```
Comment (ffoulkes) [inline docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md:262]: see earlier comment

**Review by ffoulkes — APPROVED** [review]
Aaaaa. I'm wasting my time. Management will just tell me to approve it anyway. STFP

**Review by satish153 — APPROVED** [review]
LGTM

File: docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md [hunk 6]
```diff
@@ -0,0 +1,456 @@
+<!--
+/*
```
[the remark this answers is not in the document]
Comment (n-sandeep) [reply docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md:2]: Sure, i can modify. I have re-used the comment from older file.

File: docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md [hunk 7]
```diff
@@ -0,0 +1,456 @@
...
+  ```bash
+  echo 2 > /sys/class/net/ens802f0/device/sriov_numvfs
+  ```
+
+Notes about topology:
+
+- VMs are spawned on top of each VF. Each VF will have a port representor in ACC. P4 runtime rules are configured to map VFs and their corresponding port representors.
+- Each physical port will have a port representor in ACC. P4 runtime rules are configured to map VFs and its corresponding port representors.
+- Each physical port will have a corresponding APF netdev on HOST. Create port representors in ACC for each HOST APF netdev. These APF netdev on HOST will receive unknown traffic for applications to act on.
+- All port representors should be part of an OvS bridge. Based on topology, these OvS bridges will just perform bridging or TEP termination bridges which are used to enable underlay connectivity for VxLAN traffic.
```
[the remark this answers is not in the document]
Comment (n-sandeep) [reply docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md:55]: Updated the sentense in the new patch

File: docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md [hunk 8]
```diff
@@ -0,0 +1,456 @@
...
+```bash
+# VM1 configuration
+telnet <VM1 IP> <VM1 port>
+ip link add link <Netdev connected to VF1> name <Netdev connected to VF1>.10 type vlan id 10
+ip addr add 101.0.0.1/24 dev <Netdev connected to VF1>.10
+ifconfig <Netdev connected to VF> up
+ifconfig <Netdev connected to VF>.10 up
+```
+
+Option 2: If we are unable to spawn VM's on top of the VF's, we can leverage kernel network namespaces.
```
[the remark this answers is not in the document]
Comment (n-sandeep) [reply docs/apps/lnw/es2k/es2k-lnw-underlay-ecmp.md:177]: network namespace is used to partition kernel resources, so we are giving an option to pick network kernel namespaces.

**Pull request merged**